support/oidc-discovery-provider/README.md

```diff
@@ -30,18 +30,20 @@ The provider has the following command line flags:
 The configuration file is **required** by the provider. It contains
 [HCL](https://github.com/hashicorp/hcl) encoded configurables.
 
-| Key                  | Type    | Required?   | Description                                              | Default  |
-| -------------------- | --------| ----------- | -------------------------------------------------------- | -------- |
-| `acme`               | section | required[1] | Provides the ACME configuration.                         |          |
-| `domain`             | string  | required    | The domain the provider is being served from.            |          |
-| `listen_socket_path` | string  | required[1] | Path on disk to listen with a Unix Domain Socket.        |          |
-| `log_format`         | string  | optional    | Format of the logs (either `"TEXT"` or `"JSON"`)         | `""`     |
-| `log_level`          | string  | required    | Log level (one of `"error"`,`"warn"`,`"info"`,`"debug"`) | `"info"` |
-| `log_path`           | string  | optional    | Path on disk to write the log.                           |          |
-| `log_requests`       | bool    | optional    | If true, all HTTP requests are logged at the debug level | false    |
-| `registration_api`   | section | required[2] | (Deprecated) Provides Registration API details.          |          |
-| `server_api`         | section | required[2] | Provides SPIRE Server API details.                       |          |
-| `workload_api`       | section | required[2] | Provides Workload API details.                           |          |
+| Key                     | Type    | Required?   | Description                                              | Default  |
+| ----------------------  | --------| ----------- | -------------------------------------------------------- | -------- |
+| `acme`                  | section | required[1] | Provides the ACME configuration.                         |          |
+| `domain`                | string  | required    | The domain the provider is being served from.            |          |
+| `insecure_addr`         | string  | optional    | Exposes the service on http.                             | `":80"`  |
```

Comment:
Let me know if I should leave these 2 settings undocumented.

Reply:
I am fine either way. If we do leave them documented, we should at a minimum add an annotation down below (e.g. `[2]` that explains that at least the `allow_insecure_scheme` configurable is for test purposes only and should _never_ be used in a production deployment.

@evan2645 may have stronger feelings around the documentation of these.

Reply:
added https://github.com/spiffe/spire/pull/2404/files#diff-30140922f86d99ba52fbcc88de127aa485375d6c66564a6f91af994c733e315bR52

@azdagron @evan2645